tests/test_itemized.py

```diff
@@ -249,7 +249,7 @@ def test_amount_sched_e(self):
         ]
         results = self._results(api.url_for(ScheduleEView, min_amount=100))
         self.assertTrue(all(each['expenditure_amount'] >= 100 for each in results))
-        results = self._results(api.url_for(ScheduleAView, max_amount=150))
+        results = self._results(api.url_for(ScheduleEView, max_amount=150))
```

Comment:
Don't think it is wrong, but  wondered why you changed the test from schedule A to schedule E.